### ISSUE TITLE
Fuse engaged state not restored on UI open

### ISSUE THREAD

**Issue by mliberty1** [jetperch/pyjoulescope_ui#244]
### Joulescope model

JS220

### UI version

1.0.48

### What OS are you seeing the problem on?

Windows 11

### What happened?

When a fuse is already engaged and I start the UI, the UI does not indicate that the fuse is engaged.

### What was expected?

The UI should indicate the fuse state on start.

### How to reproduce?

1. Start the Joulescope UI
2. Put 5 A through the JS220 current terminals to engage the Max Fuse
3. Observe that the UI indicates that the fuse is engaged.
4. Quit the UI
5. Start the UI
6. Observe that the UI does not indicate that the fuse is engaged.  Cannot press button to clear.

**Comment by mliberty1** [comment]
The only easy workaround while using the UI only is to unplug and replug your JS220.

Using python, you first need to know the serial number of the JS220:

```
python -m pyjoulescope_driver scan
```

Edit the following command to replace the `001707` serial number with the one from the previous scan command:
```
python -m pyjoulescope_driver set u/js220/001707 "s/fuse/+/!clear=1"
```

**Comment by mliberty1** [comment]
We found that the JS220 firmware was not correctly retaining the fuse engaged states.  When a new application connected, the firmware was not updating the host with the previous value.  The host joulescope_driver, pyjoulescope_driver, and pyjoulescope_ui all work correctly after fixing the firmware.

**Comment by mliberty1** [comment]
Fixed with JS220 FW 1.3.0, soon to be included in UI 1.2.0
